src/lib.rs

```diff
@@ -721,12 +727,13 @@ fn load_ports(target: &VcpkgTarget) -> Result<BTreeMap<String, Port>, Error> {
     // load updates to the status file that have yet to be normalized
     let status_update_dir = target.status_path.join("updates");
 
-    let paths = try!(
-        fs::read_dir(status_update_dir).map_err(|e| Error::VcpkgInstallation(format!(
-            "could not read status file updates dir: {}",
+    let paths = try!(fs::read_dir(status_update_dir.clone()).map_err(
```

Comment:
For what it's worth, I think it should work to use `&status_update_dir` here and save the clone.